x/concentrated-liquidity/model/pool.go

```diff
@@ -127,6 +133,11 @@ func (p Pool) GetTickSpacing() uint64 {
 	return p.TickSpacing
 }
 
+// GetPrecisionFactorAtPriceOne returns the precision factor at price one of the pool
+func (p Pool) GetPrecisionFactorAtPriceOne() sdk.Int {
```

Comment:
nit: unsure what the value of this getter is when the field is exported anyways

Reply:
Without this getter, how would you retrieve it from a ConcentratedPoolExtension pool object? Here is an example:

https://github.com/osmosis-labs/osmosis/blob/b021858136527902a7ad23e0693e5d88ddbc3f0f/x/concentrated-liquidity/swaps.go#L287-L290